client/src/app/site/agenda/components/agenda-list/agenda-list.component.ts

```diff
@@ -36,14 +38,17 @@ import { ViewListOfSpeakers } from '../../models/view-list-of-speakers';
 @Component({
     selector: 'os-agenda-list',
     templateUrl: './agenda-list.component.html',
-    styleUrls: ['./agenda-list.component.scss']
+    styleUrls: ['./agenda-list.component.scss'],
+    encapsulation: ViewEncapsulation.None
 })
 export class AgendaListComponent extends BaseListViewComponent<ViewItem> implements OnInit {
     /**
      * Show or hide the numbering button
      */
     public isNumberingAllowed: boolean;
 
+    public showSubtitleWithSubmitters: boolean;
```

Comment:
If option 1) is choosen, rename this to `showSubtitles`. For option 2) remove this variable.